src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -29,6 +30,13 @@ public OrdersController(IOrderRepository orderRepository)
         [Route("/api/v1/organisations/{organisationId}/[controller]")]
         public async Task<ActionResult> GetAllAsync(Guid organisationId)
         {
+            var primaryOrganisationId = User.GetPrimaryOrganisationId();
```

Comment:
Might want to change this to return a GUID as this is more inline with the type of the user's ID?